assets/src/sass/content/_search-results.scss

```diff
@@ -0,0 +1,33 @@
+// Group all style customizations for the 2019-2 transparency report together.
+.search-results {
+	&__tabs {
+		border-bottom: 1px solid #ccc;
+		margin-bottom: 2rem !important;
+		padding-bottom: 0.35rem !important;
+		padding-top: 2rem !important;
```

Comment:
Non-blocking: there's a `rem()` mixin available in the theme which can make it easier to see what these dimensions actually are in pixel units.